services/sources/publisher_backlist_service.py

```diff
@@ -96,3 +101,95 @@ def get_records_array(self,
             array_json.append(pub_backlist_records_response_json)
 
         return array_json
+    
+    def addHasPartMapping(self, resultsRecord, record):
+
+        #GOOGLE DRIVE API CALL TO GET PDF/EPUB FILES
+
+        try:
+            if 'in_copyright' in record.rights:
+                linkString = '|'.join([
+                    '1',
+                    #LINK TO PDF/EPUB,
```

Comment:
I don't think the comment is necessary here

Reply:
I put it there as a reminder to fill in that blank spot of the link_string once we have access to the PDFs/EPUBs, so I think it's fine for now.